### ISSUE TITLE
Direct passthroughs rules not removed on reload

### ISSUE THREAD

**Issue by DevDorrejo** [firewalld/firewalld#959]
**What happened**:
removing passthroughts and doing reload wont remove the rules

**What you expected to happen**:
on reload rules not appear

**How to reproduce it (as minimally and precisely as possible)**:
`firewall-cmd --permanent --direct --passthrough ipv4 -I FORWARD -o virbr0 -j ACCEPT
firewall-cmd --direct --get-all-passthroughs
firewall-cmd --permanent --direct --remove-passthrough ipv4 -I FORWARD -o virbr0 -j ACCEPT
firewall-cmd --reload
firewall-cmd --permanent --direct --remove-passthrough ipv4 -I FORWARD -o virbr0 -j ACCEPT
`

**Anything else we need to know?**:
workaround: `systemctl restart firewalld.service`

**Environment**:
- Firewalld Version (if Fedora based `dnf info firewalld` or commit hash if developing from git `git log -n1 --format=format:"%H"`): 1.1.1
- Firewalld Backend (`cat /etc/firewalld/firewalld.conf | grep FirewallBackend`): `nftables`
- OS (e.g: `cat /etc/os-release`):
```
NAME="openSUSE Tumbleweed"
# VERSION="20220428"
ID="opensuse-tumbleweed"
ID_LIKE="opensuse suse"
VERSION_ID="20220428"
PRETTY_NAME="openSUSE Tumbleweed"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:tumbleweed:20220428"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
DOCUMENTATION_URL="https://en.opensuse.org/Portal:Tumbleweed"
LOGO="distributor-logo-Tumbleweed
```
- Others:

**Comment by erig0** [comment]
Do you have `FlushAllOnReload=no` in `/etc/firewalld/firewalld.conf`?

**Comment by erig0** [comment]
tl;dr don`t use `--permanent`

You're using `--permanent`, it looks like when you use `--permanent --direct --passthrough` it synonymous with `--permanent --direct --add-passthrough`. I don't know why.

Even the man pages say you shouldn't be able to use `--permanent` with `--passthrough`:

> --direct --passthrough { ipv4 | ipv6 | eb } args
     Pass a command through to the firewall.  args can be all iptables,
     ip6tables and ebtables command line arguments. This command is
     untracked, which means that firewalld is not able to provide
     information about this command later on, also not a listing of the
     untracked passthoughs.

But, the code obviously allows it and aliases it to `--add-passthrough`:
https://github.com/firewalld/firewalld/blob/797234e232ecb680a8e65e709b04400308773ff6/src/firewall-cmd.in#L1888-L1901


We can't fix this with out potentially breaking uses that rely on `--permanent --direct --passthrough` being an alias for `--permanent --direct --add-passthrough`.

**Comment by DevDorrejo** [comment]
Thanks, understand, so i wont use `--permanent`, on this case

**Comment by erig0** [comment]
Maybe in the short term we should update the man page to indicate this is an alias for `--permanent --add-passthrough`... and that it will change in a future release, e.g. 2.0.0.

**Comment by DevDorrejo** [comment]
👍🏽
